lib/assessment.js

```diff
@@ -4,23 +4,39 @@
 // Write a function that takes zero or more arguments
 // and returns the minimum from that set
 // assume that all the argument are numbers
-var min = function min() {
+var min = function () {
+  var minimum = arguments[0]
+  for (i=1; i< arguments.length; i++){
```

Comment:
`i=1` needs a `var` to be safe! This overwrite a variable `i` in a surrounding scope, or, worst-case, create a global variable `i`!